data-raw/healthy-places/living-conditions/rough-sleeping.R

```diff
@@ -0,0 +1,95 @@
+# Installing relevant packages
+library(tidyverse)
+library(readxl)
+library(httr)
+library(dplyr)
+library(geographr)
+
```

Comment:
Does your code need all these packages? We always try to keep our scripts as efficient and concise as possible - please delete packages you are not using.